### PR TITLE
Add Haxe support

### PR DESCRIPTION
**Pull request by Gama11**
In action:

Note: as you can see, I added a new placeholder. This was necessary because Haxe does not seem to like trailing slashes in its `--cwd` argument.

### CODE REVIEW

File: src/codeManager.ts [hunk 1]
```diff
@@ -228,6 +228,13 @@ export class CodeManager {
     }
 
     /**
+     * Gets the directory of the code file without a trailing slash.
+     */
+    private getCodeFileDirWithoutTrailingSlash(): string {
+        return this.getCodeFileDir().replace(/\\$/, "")
```
Comment (formulahendry) [inline src/codeManager.ts]: Also take care of `/` on Linux/macOS

Comment (Gama11) [inline src/codeManager.ts]: Ah, good point. :)

Comment (Gama11) [inline src/codeManager.ts]: Done!

**Comment by formulahendry** [comment]
LGTM. :shipit: 
Thanks!

**Pull request merged**